test/e2e/load.go

```diff
@@ -94,17 +81,45 @@ var _ = Describe("Load capacity", func() {
 
 	type Load struct {
 		podsPerNode int
+		image       string
+		command     []string
+		skip        bool
+		deleteNs    bool
 	}
 
 	loadTests := []Load{
-		{podsPerNode: 30},
+		// The container will consume 1 cpu and 512mb of memory.
+		{podsPerNode: 3, image: "jess/stress", command: []string{"stress", "-c", "1", "-m", "2"}},
```

Comment:
How did you choose these numbers? If we are advertising 30 pods per node but can handle only 3 stress pods (which don't look very stressful, based on your -c and -m arguments) then that's not very good.